index.html

```diff
@@ -117,21 +117,31 @@ <h2>API</h2>
 import * as parsel from "./parsel.js";
 self.parsel = parsel; // so that people can experiment in the console
 
+if(location.search.includes('?selector=')) {
```

Comment:
Better use `location.searchParams`. As it currently stands, this will fail if there is any other param (and even if the website itself doesn't support other params *now*, other software often adds params).